packages/relay/tests/lib/eth.spec.ts

```diff
@@ -673,6 +753,146 @@ describe('Eth calls using MirrorNode', async function () {
     expect(result).to.equal(null);
   });
+
+  describe('eth_getLogs', async function () {
+
+    const expectLogData1 = (res) => {
+      expect(res.address).to.eq(defaultLogs.logs[0].address);
+      expect(res.blockHash).to.eq(defaultDetailedContractResults.block_hash);
+      expect(res.blockNumber).to.eq(defaultDetailedContractResults.block_number);
+      expect(res.data).to.eq(defaultLogs.logs[0].data);
+      expect(res.logIndex).to.eq(defaultLogs.logs[0].index);
+      expect(res.removed).to.eq(false);
+      expect(res.topics).to.exist;
+      expect(res.topics).to.deep.eq(defaultLogs.logs[0].topics);
+      expect(res.transactionHash).to.eq(defaultDetailedContractResults.hash);
+      expect(res.transactionIndex).to.eq(defaultDetailedContractResults.transaction_index);
+    };
+
+    const expectLogData2 = (res) => {
+      expect(res.address).to.eq(defaultLogs.logs[1].address);
+      expect(res.blockHash).to.eq(defaultDetailedContractResults.block_hash);
+      expect(res.blockNumber).to.eq(defaultDetailedContractResults.block_number);
+      expect(res.data).to.eq(defaultLogs.logs[1].data);
+      expect(res.logIndex).to.eq(defaultLogs.logs[1].index);
+      expect(res.removed).to.eq(false);
+      expect(res.topics).to.exist;
+      expect(res.topics).to.deep.eq(defaultLogs.logs[1].topics);
+      expect(res.transactionHash).to.eq(defaultDetailedContractResults.hash);
+      expect(res.transactionIndex).to.eq(defaultDetailedContractResults.transaction_index);
+    };
+
+    const expectLogData3 = (res) => {
+      expect(res.address).to.eq(defaultLogs.logs[2].address);
+      expect(res.blockHash).to.eq(defaultDetailedContractResults2.block_hash);
+      expect(res.blockNumber).to.eq(defaultDetailedContractResults2.block_number);
+      expect(res.data).to.eq(defaultLogs.logs[2].data);
+      expect(res.logIndex).to.eq(defaultLogs.logs[2].index);
+      expect(res.removed).to.eq(false);
+      expect(res.topics).to.exist;
+      expect(res.topics).to.deep.eq(defaultLogs.logs[2].topics);
+      expect(res.transactionHash).to.eq(defaultDetailedContractResults2.hash);
+      expect(res.transactionIndex).to.eq(defaultDetailedContractResults2.transaction_index);
+    };
+
+    const expectLogData4 = (res) => {
+      expect(res.address).to.eq(defaultLogs.logs[3].address);
+      expect(res.blockHash).to.eq(defaultDetailedContractResults3.block_hash);
+      expect(res.blockNumber).to.eq(defaultDetailedContractResults3.block_number);
+      expect(res.data).to.eq(defaultLogs.logs[3].data);
+      expect(res.logIndex).to.eq(defaultLogs.logs[3].index);
+      expect(res.removed).to.eq(false);
+      expect(res.topics).to.exist;
+      expect(res.topics).to.deep.eq(defaultLogs.logs[3].topics);
+      expect(res.transactionHash).to.eq(defaultDetailedContractResults3.hash);
+      expect(res.transactionIndex).to.eq(defaultDetailedContractResults3.transaction_index);
+    };
+
+    it('no filters', async function () {
```

Comment:
Please add a no match case and a case where mirrorNode response is 400 or 404 to ensure `eth.getLogs` handles the response well